experts-migration-commands.php

```diff
@@ -0,0 +1,742 @@
+	
+		$gutenberg_template = '<!-- wp:columns -->
+	<div class="wp-block-columns"><!-- wp:column {"width":"33.33%%"} -->
+	<div class="wp-block-column" style="flex-basis:33.33%%">
+	<!-- wp:image {"id":10000,"sizeSlug":"full","linkDestination":"none"} -->
+	<figure class="wp-block-image size-full"><img src="%s" alt="" class="wp-image-10000"/></figure>
+	<!-- /wp:image --></div>
+	<!-- /wp:column -->
+	
+	<!-- wp:column {"width":"66.66%%"} -->
+	<div class="wp-block-column" style="flex-basis:66.66%%"><!-- wp:heading -->
+	<h2>%s</h2>
+	<!-- /wp:heading -->
+	
+	<!-- wp:heading {"level":3} -->
+	<h3>%s</h3>
+	<!-- /wp:heading -->
+	
+	<!-- wp:heading {"level":4} -->
+	<h4>%s</h4>
+	<!-- /wp:heading -->
+	
+	<!-- wp:heading {"level":4} -->
+	<h4>%s</h4>
+	<!-- /wp:heading -->
+	
+	<!-- wp:heading {"level":4} -->
+	<h4>%s</h4>
+	<!-- /wp:heading -->
+	
+	<!-- wp:heading {"level":3} -->
+	<h3><br>Fields</h3>
+	<!-- /wp:heading -->
+	
+	<!-- wp:paragraph -->
+	<p>%s</p>
+	<!-- /wp:paragraph -->
+	
+	<!-- wp:heading {"level":3} -->
+	<h3><br>Expertise</h3>
+	<!-- /wp:heading -->
+	
+	<!-- wp:paragraph -->
+	<p>%s</p>
+	<!-- /wp:paragraph -->
+	
+	
+	</div>
+	<!-- /wp:column --></div>
+	<!-- /wp:columns -->
+	
+	<!-- wp:columns -->
+	<div class="wp-block-columns">
+	<!-- wp:column {"width":"33.33%%"} -->
+	<div class="wp-block-column" style="flex-basis:33.33%%">
+	<!-- wp:heading {"level":3} -->
+	<h3>Contact Information</h3>
+	<!-- /wp:heading -->
+	%s 
+	
+	%s 
+	
+	%s
+	
+	%s
+	
+	%s
+	
+	%s
+	
+	<!-- wp:block {"ref":%d} /-->
+	</div>
+	<!-- /wp:column -->
+	
+	<!-- wp:column {"width":"66.66%%"} -->
+	<div class="wp-block-column" style="flex-basis:66.66%%">
+	
+	%s
+	
+	%s 
+	
+	%s
+	
+	%s
+	</div>
+	<!-- /wp:column -->
+	</div>
+	<!-- /wp:columns -->';
+	
+		$paged = ( $args[0] ) ? $args[0] : 1;
+	
+		$profile_args = array(
+			'post_type'			=> 'profile_cct',
+			'posts_per_page'    => 30,
+			'paged' 			=> $paged
+		);
+		
+		$profiles = new WP_Query($profile_args);
+		if ( $profiles->have_posts() ) {
+	
+			// wp_block
+			while( $profiles->have_posts() ){
+				$profiles->the_post();
+				$post_id = get_the_ID();
+				// taxonomies
+				$faculties = get_the_terms( $post_id, 'profile_cct_faculty' );
+				$campuses = get_the_terms( $post_id, 'profile_cct_campus' );
+				$fields = get_the_terms( $post_id, 'profile_cct_field' );
+				$faculty_links = $this->ubc_experts_process_terms($faculties);
+				$campus_links = $this->ubc_experts_process_terms($campuses);
+				$field_links = $this->ubc_experts_process_terms($fields);
+	
+				//meta
+				$profile_meta = get_post_meta($post_id, 'profile_cct', true);
+				$profile_pic = get_post_meta($post_id, '_thumbnail_id', true);
+				$profile_pic_url = wp_get_attachment_image_url($profile_pic, 'full');
+				$name_keys = array('salutations','first','middle','last','credentials' );
+				$name = $this->ubc_experts_process_array_keys($profile_meta['name'], $name_keys, '', ' ');
```

Comment:
May want to ensure the array key exists with a non-empty value before using it here (and elsewhere in this while block).